markup/house-style.md

```diff
@@ -75,6 +76,55 @@ We *don't* do this:
 ```html
 <meta charset="UTF-8" />
 ```
+### Form submit controls
+
+On modern browsers, `button type="submit"` provides a superset of functionality when compared to `input type=submit`.
+
+`button` elements are much easier to style than `input` elements. You can add
+inner HTML content (think `<em>`, `<strong>`, or even `<img>`), and use
+`::after` and `::before` pseudo-elements to achieve complex rendering while
+`input` only accepts a textual value attribute.
+
+Make sure to specify `name` and `value` attributes to maximize browsers
```

Comment:
How does this maximize browser compatibility? Is it possible to explain, or to link out to something?